server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -143,6 +158,25 @@ export class RwcdsFormService {
       &$expand=dcp_ZoningResolution
     `);
 
+    // dcp_includezoningtextamendment is a field on the rwcds-form that should be "Yes" (717170000) if...
+    // there exists a "ZR" action and that action's dcp_ZoningResolution.dcp_zoningresolution is "AppendixF"
+    const includeZoningTextAmendmentAction = projectActions.find(action =>
+      action['_dcp_action_value@OData.Community.Display.V1.FormattedValue'] === 'ZR'
+      && action.dcp_ZoningResolution.dcp_zoningresolution === 'AppendixF'
```

Comment:
I'm sorry but could we move this into another method?

It could be a private method and `syncActions` calls it.

Splitting things out into new methods gives you a way to name that chunk of logic in a way that makes it clearer (and self-documents). It makes it easier to debug. It also keeps the method shorter.

Please think about the method as a whole and what it's doing — is the method concise? Does it do too much? Should we keep adding on to the amount it's doing? Etc.

Reply:
Yep, I'll move